src/components/PostList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CommentList from './CommentList';
+import Post from './Post';
+import comments from '../api/comments';
+
+function PostList({
+  posts,
+}) {
+  return (
+    posts.map(elem => (
+      <div className="container">
+        <Post {...elem} />
+        <CommentList coments={comments} currentPost={elem} />
```

Comment:
```suggestion
        <CommentList coments={post.comments} currentPost={elem} />
```
find needed comments in App.js, where you find needed user